transcribe/transcribe.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2019 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pathlib
+import tempfile
+from typing import Literal, Tuple, Union
+
+import mlrun
+import numpy as np
+import pandas as pd
+import whisper
+from mutagen.mp3 import MP3
+from tqdm.auto import tqdm
+
+
+def transcribe(
+    context: mlrun.MLClientCtx,
+    audio_files_directory: str,
```

Comment:
Better to have like an `input_path` as a parameter instead of `audio_files_directory` so this function can run both on directories and on single files. Remember that each function is usually being run at first locally so a user might want to first test it out on a single file just to see what it is doing.

You can adjust the code to work on single file or on multiple files very easily. If its a dir, collect files into a list as it is doing now, if its a file, wrap it in a list and continue.

Reply:
It is already like this, just forgot to update this parameter